crates/astria-sequencer/src/fees/action.rs

```diff
@@ -137,98 +137,290 @@ impl ActionHandler for FeeAssetChange {
+    async fn ibc_relay_fee_change_action_executes_as_expected() {
+        let (initial_fees, initial_fee_change, new_fees, new_fee_change) =
+            get_default_fees_and_fee_changes!(IbcRelay);
+        test_fee_change_action::<IbcRelay, _>(
+            initial_fees,
+            initial_fee_change,
+            new_fees,
+            new_fee_change,
+            "ibc_relay",
+        )
+        .await;
+    }
+
+    #[tokio::test]
+    async fn ibc_relayer_change_fee_change_action_executes_as_expected() {
+        let (initial_fees, initial_fee_change, new_fees, new_fee_change) =
+            get_default_fees_and_fee_changes!(IbcRelayerChange);
+        test_fee_change_action::<IbcRelayerChange, _>(
+            initial_fees,
+            initial_fee_change,
+            new_fees,
+            new_fee_change,
+            "ibc_relayer_change",
+        )
+        .await;
+    }
+
+    #[tokio::test]
+    async fn fee_asset_change_fee_change_action_executes_as_expected() {
+        let (initial_fees, initial_fee_change, new_fees, new_fee_change) =
+            get_default_fees_and_fee_changes!(FeeAssetChange);
+        test_fee_change_action::<FeeAssetChange, _>(
+            initial_fees,
+            initial_fee_change,
+            new_fees,
+            new_fee_change,
+            "fee_asset_change",
+        )
+        .await;
+    }
+
+    #[tokio::test]
+    async fn fee_change_fee_change_action_executes_as_expected() {
+        let (initial_fees, initial_fee_change, new_fees, new_fee_change) =
+            get_default_fees_and_fee_changes!(FeeChange);
+        test_fee_change_action::<FeeChange, _>(
+            initial_fees,
+            initial_fee_change,
+            new_fees,
+            new_fee_change,
+            "fee_change",
+        )
+        .await;
+    }
+
+    #[tokio::test]
+    async fn sudo_address_change_fee_change_action_executes_as_expected() {
+        let (initial_fees, initial_fee_change, new_fees, new_fee_change) =
+            get_default_fees_and_fee_changes!(SudoAddressChange);
+        test_fee_change_action::<SudoAddressChange, _>(
+            initial_fees,
+            initial_fee_change,
+            new_fees,
+            new_fee_change,
+            "sudo_address_change",
+        )
+        .await;
+    }
+
+    #[tokio::test]
+    async fn ibc_sudo_change_fee_change_action_executes_as_expected() {
+        let (initial_fees, initial_fee_change, new_fees, new_fee_change) =
+            get_default_fees_and_fee_changes!(IbcSudoChange);
+        test_fee_change_action::<IbcSudoChange, _>(
+            initial_fees,
+            initial_fee_change,
+            new_fees,
+            new_fee_change,
+            "ibc_sudo_change",
+        )
+        .await;
+    }
+
+    async fn test_fee_change_action<F: FeeHandler<FeeComponents = T>, T>(
+        initial_fees: T,
+        initial_fee_change: FeeChange,
+        new_fees: T,
+        new_fee_change: FeeChange,
+        action_name: &str,
```

Comment:
not sure using `action_name` is super useful here, as if the test fails, it'll print the name of the test which has the action name

Reply:
Removed in 3a71450